productService/modules/createProduct.ts

```diff
@@ -0,0 +1,57 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import { createResponse } from '../helpers/createResponse';
+import { Product } from '../models/Product';
+import { PRODUCT_QUERY } from '../queries/index';
+import connectToDb from '../helpers/connectToDb';
+
+export const createProduct: APIGatewayProxyHandler = async (event) => {
+  const client = await connectToDb();
+
+  try {
+    console.log('create product: ', event);
+
+    await client.query('BEGIN');
+
+    const body = JSON.parse(event.body) as Product;
+    const { title, description, price, imagelink } = body;
+    console.log(`method createProduct was called with: title ${title}; description ${description}; price ${price}; imagelink ${imagelink}`);
+
+    const insertIntoProductQuery = {
+      text: PRODUCT_QUERY.INSERT_INTO_PRODUCTS,
+      values: [title, description, price, imagelink]
+    }
+
+    const { rows } = await client.query(insertIntoProductQuery);
+    const productId = rows[0].id;
+
+    const insertIntoStocksQuery = {
+      text: PRODUCT_QUERY.INSERT_INTO_STOCKS,
+      values: [productId, 1]
```

Comment:
Hardcoded `count` value. You should take `count` from request.

Reply:
Thank you! Fixed now.